tests/common/mod.rs

```diff
@@ -0,0 +1,44 @@
+use bom;
+use bom::bom::options::{BitcodeOptions, ExtractOptions, Options, Subcommand};
+use std::path::{PathBuf};
+
+pub static SOURCE_DIR: &'static str = "tests/sources";
+
+
+// Get the clang command provided by the user (via the CLANG environment variable), if any
+//
+// If the user did not provide one, just return None, which is interpreted by build-bom as 'clang'
+pub fn user_clang_cmd() -> Option<PathBuf> {
+    std::env::var("CLANG").map(|s| { let mut p = std::path::PathBuf::new(); p.push(s); p }).ok()
+}
+
+// Get the user-provided llvm-link command (via the LLVM_LINK environment variable), if any
+//
+// If the user did not provide one, return None, which build-bom interprets as 'llvm-link'
+pub fn user_llvm_link_cmd() -> Option<String> {
+    std::env::var("LLVM_LINK").ok()
+}
+
+// Get the user-provided llvm-dis command (via the LLVM_DIS environment variable), if any
+//
+// If the user did not provide one, return None, which build-bom interprets as 'llvm-dis'
+#[allow(dead_code)]
```

Comment:
Is this directive necessary? It looks like it is used later

Reply:
Each test file becomes a separate crate, so while this is used in test_zlib.rs, it isn't used in test_bom.rs, so it generates a warning during the independent compile of the latter without this directive.

Reply:
I suppose I could move it into test_zlib.rs, but that would move it away from the other two similar lookups.  Let me know which you prefer.

Reply:
Weird. It would be worth a note there at least. No need for big changes.